kortex_driver/src/hardware_interface.cpp

```diff
@@ -810,14 +810,22 @@ void KortexMultiInterfaceHardware::readGripperPosition()
 }
 
 return_type KortexMultiInterfaceHardware::write(
-  const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/)
+  const rclcpp::Time & /*time*/, const rclcpp::Duration & period)
 {
   if (block_write)
   {
     feedback_ = base_cyclic_.RefreshFeedback();
     return return_type::OK;
   }
 
+  const auto max_period = 0.002;
```

Comment:
Any chance that the hardware_interface knows the desired update rate? If so, we could remove the hard coded value for double the desired period.